samples/subsys/usb/cdc_acm_perf/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _usb_cdc-acm:
+
+USB CDC ACM Peformance Sample Application
+#########################################
+
+Overview
+********
+
+This sample tests the performance of the USB Communication Device Class (CDC)
+Abstract Control Model (ACM) driver provided by the Zephyr project.
+This sample can be found under :zephyr_file:`samples/subsys/usb/cdc_acm_perf` in the
+Zephyr project tree.
+
+This sample firmware streams incrementing float32 data out the serial port as fast as possible.
```

Comment:
`float32` does not make sense and as far as I checked the code you just use `uint32_t`.

Reply:
you are correct, thanks for pointing that out.